src/main/java/org/launchcode/techjobs/oo/Main.java

```diff
@@ -2,21 +2,46 @@
 
 import java.util.ArrayList;
 
+//public class Main {
+//
+//    public static void main(String[] args) {
+////        Job job1 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+////        Job job2 = new Job("Web Developer", new Employer("LaunchCode"), new Location("St. Louis"), new PositionType("Front-end developer"), new CoreCompetency("JavaScript"));
+////        Job job3 = new Job("Ice cream tester", new Employer(""), new Location("Home"), new PositionType("UX"), new CoreCompetency("Tasting ability"));
+////
+////        ArrayList<Job> jobs = new ArrayList<>();
+////        jobs.add(job1);
+////        jobs.add(job2);
+////        jobs.add(job3);
+////
+////        for (Job job : jobs){
+////            System.out.println(job);
+////        }
+//    }
+//
+//}
+import java.util.ArrayList;
+
 public class Main {
 
     public static void main(String[] args) {
-//        Job job1 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
-//        Job job2 = new Job("Web Developer", new Employer("LaunchCode"), new Location("St. Louis"), new PositionType("Front-end developer"), new CoreCompetency("JavaScript"));
-//        Job job3 = new Job("Ice cream tester", new Employer(""), new Location("Home"), new PositionType("UX"), new CoreCompetency("Tasting ability"));
-//
-//        ArrayList<Job> jobs = new ArrayList<>();
-//        jobs.add(job1);
-//        jobs.add(job2);
-//        jobs.add(job3);
-//
-//        for (Job job : jobs){
-//            System.out.println(job);
-//        }
+        Employer acme = new Employer("ADME");
+        Job job1 = new Job("Product tester", acme, new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+        Job job2 = new Job("Web Developer", acme, new Location("St. Louis"), new PositionType("Front-end developer"), new CoreCompetency("JavaScript"));
+        Job job3 = new Job("Ice cream tester", new Employer(""), new Location("Home"), new PositionType("UX"), new CoreCompetency("Tasting ability"));
+
+        ArrayList<Job> jobs = new ArrayList<>();
+        jobs.add(job1);
+        jobs.add(job2);
+        jobs.add(job3);
+
+        for (Job job : jobs){
+            System.out.println(job);
+        }
+        acme.setValue("ACME");
+        for (Job job : jobs){
+            System.out.println(job);
+        }
```

Comment:
Are these lines for testing? If so, you may need to move them into your `JobTest` file instead of as a scenario to test.
```suggestion
```